frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.tsx

```diff
@@ -123,6 +123,8 @@ export function LayoutMenu(props: LayoutMenuProps) {
     autoSaveLayouts,
     setAutoSaveLayouts,
     saveCurrentLayout,
+    // rome-ignore lint/correctness/noUnusedVariables: underscore prefix does not work with object destructuring
+    setCurrentLayout,
```

Comment:
fix for accidentally introduced bug in my recent Linter PR

Reply:
React/Antd complaining about the `setCurrentLayout` prop being present in/leaking into a `<Menu>` component, where it was not expected/supported.